moninag/tests/unittests/nagplugin/test_view.py

```diff
@@ -0,0 +1,48 @@
+import json
```

Comment:
Unused import :upside_down_face: 
(but maybe it will be necessary, look next comment) :next_track_button: